### DISCUSSION TITLE
Linux环境下状态栏无法正确显示选中

### DISCUSSION THREAD

**Discussion by Rose-xiao** [libnyanpasu/clash-nyanpasu#550] [Q&A]
如图，我以选中了全局代理，但是在图标的状态栏里没有任何反馈

![截图 2024-03-04 16-55-42](https://github.com/LibNyanpasu/clash-nyanpasu/assets/90542586/5c488838-2cd3-463b-95f6-49fedca5b3a5)

**Comment by keiko233** [comment]
需要提供日志，系统，

**Reply by Rose-xiao** [reply]
不会呀、以前使用的是clash for Windows的cfw、那时好像还能显示，然后使用clash-verge的时候状态好像就不再显示了、现在使用clash-nyanpasu以为会解决这个问题，但似乎不行，不过节点选择那块儿有选中显示

**Reply by NalCol** [reply]
cfw用的框架都和verge,nyanpasu都不一样，Linux需要针对性适配的😂
